libcollector/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>libcollector</name>
+  <version>0.0.0</version>
+  <description>Aggregation utilities to measure and publish metrics.</description>
```

Comment:
Is this description still correct?

Reply:
Will update. Good catch on the library name, maybe ```libstatisticscollector```

Reply:
Renamed library